modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -77,6 +79,20 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+        HttpServletRequest httpServletRequest, String name) {
+
+        CookieUtil.get(httpServletRequest, "test123");
+        CookiesManagerUtil.getCookieValue(name, httpServletRequest);
+    }
```

Comment:
@MicaelleSilvaa, source format incorrect again :disappointed: [2]

Did you run source format on this code?

If you're facing difficulties please let me know :wink:

Reply:
sorry, I ran the source format in the code, I think it's correct now.